### DISCUSSION TITLE
Set some graphic / prototype submission guidelines

### DISCUSSION THREAD

**Discussion by Bosch-0** [BitcoinDesign/Guide#254] [General]
It would be worth setting some guidelines for submitting images / prototypes to keep GH management clean. Maybe this could be included in the guide somewhere and/or GH. Here is some rough guidelines, I know this would personally help me out figuring out what to focus on:

1. Graphic / prototype (G/P) request is made via an issue. Title examples: 'Prototype request: Protecting a wallet.' or 'Graphic request: Protecting a wallet banner image.'
    - We could make issue templates for both graphics and prototypes. 
2. 'A' designs a G/P and shares it in the original issue as a screenshot / link to Figma etc.
3. 'B' has another design idea so they submit their own, also in the original issue.
4. Community decides which is most appropriate / changes made based on feedback. 
    - This should also be cross shared on Slack / other relevant channels for feedback. 
5. Once an G/P is decided on in the issue only then is a PR open for a final live review. 

I'd also suggest having a Prototype Guidelines included in the Guide on how these should be presented UI/UX wise as well as include details about submitting prototypes. Maybe we could have a 'Guide prototypes' file in the BDC Figma community profile that is home to all the live prototypes in the Guide once we have this setup. Designers would obviously have to give their designs over to whoever manages the BDC community account but it would be better to have all prototypes in one place (what if I include a prototype in the Guide then delete that file for example?). 

Food for thought!

**Comment by pavlenex** [comment]
Converted to discussion since it's not really an `issue`.

**Reply by Bosch-0** [reply]
Ty, the overall point is that it would be good to atomize tasks if we can, makes it easier for people to jump in and do something. Can be daunting when someone shares a whole page looking for feedback and there is various issues that could be tackled individually within the page.

I also really want to submit some prototypes but don't feel too comfortable starting work on one without some rough guidelines on styling

**Comment by Bosch-0** [comment]
Relevant issue for prototypes https://github.com/BitcoinDesign/Guide/issues/315
